examples/http_config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Let's make this file more generic since hopefully we can get these changes merged into the upstream respository.